docs/docs/reference/language-guide/feel-boolean-expressions.md

```diff
@@ -230,6 +230,12 @@ Use the type `Any` to check if the value is not `null`.
 
 null instance of Any
 // false
+
+"duration("P3M") instance of years and months duration"
+// true
+
+"duration("PT4H") instance of days and time duration"
+// true
```

Comment:
:x: The expression is wrapped into double quotes.

```suggestion
duration("P3M") instance of years and months duration
// true

duration("PT4H") instance of days and time duration
// true
```

Reply:
:x: We should update the type names in the description above. 

We can replace the old names for the duration types. A user should use the official type names.